.github/workflows/CI.yml

```diff
@@ -12,20 +12,29 @@ jobs:
       matrix:
         os: [windows-latest, ubuntu-latest]
     runs-on: ${{matrix.os}}
-    env:
-      buildVersion: 0.0.0.0 #The initial build version, calculated and updated in the script below
-      ACTIONS_ALLOW_UNSECURE_COMMANDS: true #Allow us to set an environment variable in the PowerShell script to set the package version (technically deprecated)
     steps:
-    - uses: actions/checkout@v1
-    - name: Custom script to create a unique application version number for this build
-      run: src/BuildVersion.ps1 -ProjectFile "src/AzurePipelinesToGitHubActionsConverter.Core/AzurePipelinesToGitHubActionsConverter.Core.csproj"
-      shell: pwsh
-    - name: Build with dotnet
-      run: dotnet build src/AzurePipelinesToGitHubActionsConverter.Core/AzurePipelinesToGitHubActionsConverter.Core.csproj --configuration Release
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 #fetch-depth is needed for GitVersion
+        
+    #Install and calculate the new version with GitVersion  
+    - name: Install GitVersion
+      uses: gittools/actions/gitversion/setup@v0.9.7
+      with:
+        versionSpec: '5.x'
+    - name: Determine Version
```

Comment:
just a habit from azdo pipielines, would it not be 'best practise' to have the '- uses: ...' as always the first line of the action (as it's less likely to be customisable) and the 'name: ...' to be the 2nd line?  Also better practise to split all actions with 1 empty line for readability?